packages/espree/lib/espree.js

```diff
@@ -12,15 +83,17 @@ const ESPRIMA_FINISH_NODE = Symbol("espree's esprimaFinishNode");
  * Converts an Acorn comment to a Esprima comment.
  * @param {boolean} block True if it's a block comment, false if not.
  * @param {string} text The text of the comment.
- * @param {int} start The index at which the comment starts.
- * @param {int} end The index at which the comment ends.
- * @param {Location} startLoc The location at which the comment starts.
- * @param {Location} endLoc The location at which the comment ends.
- * @param {string} code The source code being parsed.
- * @returns {Object} The comment object.
+ * @param {Integer} start The index at which the comment starts.
+ * @param {Integer} end The index at which the comment ends.
+ * @param {acorn.Position | undefined} startLoc The location at which the comment starts.
+ * @param {acorn.Position | undefined} endLoc The location at which the comment ends.
+ * @param {string | string} code The source code being parsed.
```

Comment:
Maybe an accidental change?

```suggestion
 * @param {string} code The source code being parsed.
```